README.md

```diff
@@ -44,29 +36,38 @@ So that I can better appreciate the context of a peep
 I want to see the time at which it was made
 ```
 
-Notes on functionality:
-------
 
-* Drive the creation of your app using tests - either cucumber or rspec as you prefer
-* Makers sign up to chitter with their email, password, name and a user name (e.g. sam@makersacademy.com, s3cr3t, Samuel Russell Hampden Joseph, tansaku).
-* The username and email are unique.
-* Peeps (posts to chitter) have the name of the maker and their user handle.
-* Use bcrypt to secure the passwords.
-* Use data mapper and postgres to save the data.
-* You don't have to be logged in to see the peeps.
-* You only can peep if you are logged in.
-* Please ensure that you update your README to indicate the technologies used, and give instructions on how to install and run the tests
-* Finally submit a pull request before Monday at 9am with your solution or partial solution.  However much or little amount of code you wrote please please please submit a pull request before Monday at 9am
+## Technologies Used
+- Ruby
+- Rspec
+- Capybara
+- Sinatra
+- Postgres
+- DataMapper
 
-Bonus:
------
 
-If you have time you can implement the following:
+To run:
 
-* In order to start a conversation as a maker I want to reply to a peep from another maker.
+In order to run tests, fork/clone this repo and run rspec in the command line. Ensure that the capybara gem is added to your Gemfile. To run the app fork the repo and in the command line: run rackup in your local directory or run it directly (ruby chitter.rb)
 
-And/Or:
 
-* Work on the css to make it look good (we all like beautiful things).
+## Job List
+* Test drive creation of app using rspec (kid of did this anyways)
+* Create list of posts/peeps that can be viewed in chronological order - most recent first
+* Create sign up form requesting user to sign up by entering their email address, name, a username, password and password confirmation in the sign up form (username and email must be unique)
+* Create a log in form for those users that have already signed up - containing just username and password input fields
+* Use bcrypt to secure the passwords
+* Once you log in you can post a message to/on Chitter
 
-Good luck and let the chitter begin!
+
+
+## Still to complete/refactor
+* Fix configuration issue/database issue - no data is being added to databases anymore - datamapper / postgres config issue?
+* Add posts to /post page
+* Show who has posted the message and at what time
+* Add additional styling
+* Peeps (posts to chitter) have the name of the maker and their user handle, as well as time peep was posted
+* If you have time you can implement the following:
+
```

Comment:
config seems to be working. can add posts to posts page.

Can add the rest later.